contrib/message-capture/message-capture-parser.py

```diff
@@ -122,19 +122,20 @@ def process_file(path: str, messages: List[Any], recv: bool, progress_bar: Optio
             msg_ser = BytesIO(f_in.read(length))
 
             # Determine message type
-            if msgtype not in MESSAGEMAP:
-                # Unrecognized message type
+            if msgtype not in MESSAGEMAP or MESSAGEMAP[msgtype] is None:
+                # Unrecognized or unhandled message type
                 try:
                     msgtype_tmp = msgtype.decode()
                     if not msgtype_tmp.isprintable():
                         raise UnicodeDecodeError
                     msg_dict["msgtype"] = msgtype_tmp
                 except UnicodeDecodeError:
                     msg_dict["msgtype"] = "UNREADABLE"
+                err_str = "Unrecognized" if msgtype not in MESSAGEMAP else "Unhandled"
                 msg_dict["body"] = msg_ser.read().hex()
-                msg_dict["error"] = "Unrecognized message type."
+                msg_dict["error"] = f"{err_str} message type."
```

Comment:
```suggestion
                msg_dict["error"] = f"{err_str} message type"
```

Reply:
Oops. I guess we don't want a period there. I thought it was a typo at first.